addons/pos_restaurant/static/src/app/control_buttons/control_buttons.js

```diff
@@ -28,7 +28,7 @@ patch(ControlButtons.prototype, {
             title: _t("Guests?"),
             feedback: (buffer) => {
                 const value = this.env.utils.formatCurrency(
-                    this.currentOrder.amountPerGuest(parseInt(buffer, 10) || 0)
+                    this.currentOrder?.amountPerGuest(parseInt(buffer, 10) || 0) || 0
```

Comment:
Is it possible for amountPerGuest to return something other than a number?

Reply:
In one way or another, I had this.currentOrder equals to undefined at some point. As formatCurrency does not handle undefined, we need a || 0 at the end of the line